dao-contracts/src/redistribution/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod cspr;
```

Comment:
I wouldn't create another module and keep redistribution functions in the `redistribution`  module. If we had any other redistribution like reputation, an additional module would be ok.